README.md

```diff
@@ -18,6 +18,18 @@ Export your GitHub API token:
 
     export GITHUB_API_TOKEN="your-organization-github-token-here"
 
+### Github Omniauth Integration:
+     
+    You must create a developer application to use omniauth.
+    If you do NOT add the appropriate env keys below AUTH will be OFF I.E => Fail unsafe. Please be carefull and check the logs. 
```

Comment:
Perhaps it is a term from my control system days http://en.wikipedia.org/wiki/Fail-safe ( opposite ). I'll rewrite it..